ddtrace/internal/processor/profiling.py

```diff
@@ -0,0 +1,44 @@
+import typing
+
+import attr
+
+from ddtrace.ext import SpanTypes
+from ddtrace.internal import nogevent
+from ddtrace.internal.compat import ensure_str
+from ddtrace.internal.processor import SpanProcessor
+from ddtrace.span import Span
+
+
+EndpointCountsType = typing.Dict[str, int]
+
+
+@attr.s(eq=False)
+class ProfilingSpanProcessor(SpanProcessor):
```

Comment:
Whilst this is used (primarily) by the profiler, I think perhaps `EndpointCallCounterProcessor` might be more appropriate to describe what this processor is doing. WDYT?

Reply:
Renamed